end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -183,6 +183,46 @@ describe('study view screenshot test', function(){
     });
 });
 
+describe("coexpression tab screenshot tests", function() {
+    before(function() {
+        var url = `${CBIOPORTAL_URL}/index.do?tab_index=tab_visualize&cancer_study_list=coadread_tcga_pub&cancer_study_id=coadread_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&Z_SCORE_THRESHOLD=2.0&case_set_id=coadread_tcga_pub_nonhypermut&case_ids=&gene_list=KRAS+NRAS+BRAF&gene_set_choice=user-defined-list&Action=Submit#coexp`;
+        goToUrlAndSetLocalStorage(url);
+    });
+    it('coexpression tab coadread_tcga_pub initial load', function() {
```

Comment:
add skip and log issue to turn these on

Reply:
how come?

Reply:
impossible to create screenshots on circleCI until backend changes deployed to cbioportal.org